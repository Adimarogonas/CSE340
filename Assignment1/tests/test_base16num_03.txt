1ABCGx16

1x16
