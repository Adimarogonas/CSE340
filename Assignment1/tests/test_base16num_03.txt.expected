{1 , NUM , 1}
{ABCGx16 , ID , 1}
{1x16 , BASE16NUM , 3}
{ , END_OF_FILE , 5}
